Documentation/contributing/development/reviewers_committers/review_docs.rst

```diff
@@ -0,0 +1,228 @@
+directives, or work on tables or tabs. Also look at the preview if you have a
+doubt as to the validity of the reStructuredText (RST) mark-up that the author
+uses.
+
+Formatting
+----------
+
+Read :ref:`Cilium's documentation style guide <docs_style_guide>`.
+
+Flag poor formatting or obvious mistakes. The syntax for RST is not always
+trivial and some contributors get it wrong, or they simply forget this is RST
+and they employ Markdown mark-up instead. Make sure they fix such issues.
+
+Keep an eye on :ref:`code-blocks <docs_style_code_blocks>`: do they include RST
+substitutions, and if so, do they use the right directive? If not, do they use
+the right language?
+
+Beyond that, the amount of time you should spend on suggestions for improving
+formatting is up to you.
+
+Grammar and style
+-----------------
+
+Read :ref:`Cilium's documentation style guide <docs_style_guide>`.
+
+Flag obvious grammar mistakes. Try to read the updated text as a user would.
+Ask the contributors to rework any sentence that is too difficult to read or to
+understand.
+
+We aim at keeping the documentation clean, consistent, and in a clear and
+comprehensible state. User experience must always be as good as possible. To
+achieve this objective, Documentation updates should follow best practices,
+such as the ones from the style guide, but reviewing PRs in-depth to flag all
+potential style improvements can be time consuming, so the amount of effort
+that you put into style guidance is up to you.
+
+As of this writing, there is no tooling in place to enforce particular style
+recommendations.
+
+Documentation build
+===================
+
+The build framework
+-------------------
+
+To do. See:
+
+  - :ref:`Instructions for building the documentation locally
+    <testing-documentation>`
+  - ``Documentation/Makefile``, ``Documentation/Dockerfile``,
+    ``Documentation/check-build.sh``
+  - Dependencies are in ``Documentation/requirements.txt``, which is generated
+    from ``Documentation/requirements_min/requirements.txt``
+  - The Sphinx theme we use is `our own fork <cilium_rtd_theme_>`_ of Read the
+    Docs's theme
+
+.. _cilium_rtd_theme: https://github.com/cilium/sphinx_rtd_theme
+
+Relevant CI workflows
+---------------------
+
+Netlify preview
+~~~~~~~~~~~~~~~
+
+Documentation changes trigger the build of a new Netlify preview. If the build
+fails, the PR authors or reviewers must investigate it (ideally the author, but
+in practice, contributors are not always familiar with RST or with our build
+framework, so consider giving a hand).
+
+Documentation build
+~~~~~~~~~~~~~~~~~~~
+
+Same as the Netlify preview, the Documentation workflow runs on doc changes and
+can raise missing updates on various generated pieces of documentation.
+
+Checkpatch
+~~~~~~~~~~
+
+The Checkpatch workflow (part of the BPF tests) is not directly relevant to
+documentation, but may raise some patch formatting issues, so you should run it
```

Comment:
It might make sense to add information on what a patch formatting issue is, but definitely not necessary.

Reply:
It's “commit subject is too long”.

... unless you start touching the `bpf/` code.

I'll figure out something.